README.md

```diff
@@ -34,51 +39,116 @@ We verify this package against N-2 recent versions of Go compiler. As of Septemb
 $ go get github.com/getsentry/sentry-go
 ```
 
-Or, if you are already using Go Modules, specify a version number as well:
+Or, if you are already using
+[Go Modules](https://github.com/golang/go/wiki/Modules), you may specify a
+version number as well:
 
 ```bash
-$ go get github.com/getsentry/sentry-go@v0.3.0
```

Comment:
Again this is to simplify maintenance and avoid inducing users to install an old version.